projects/elife.yaml

```diff
@@ -210,6 +210,8 @@ defaults:
             # two subnets in two different availability zones, both used in master nodes and in worker nodes
             subnet-id: subnet-0c60bc3cb24a2816b
             redundant-subnet-id: subnet-009a04cbefc8e3661
+            worker-subnet-id: subnet-0c60bc3cb24a2816b
+            worker-redundant-subnet-id: subnet-009a04cbefc8e3661
```

Comment:
so I guess the idea is to introduce the concept of a second pair of subnets for separate purposes that, by default, are the same as the subnet and redundant-subnet, then update a cluster instance once the new subnets are in place?